Cargo.toml

```diff
@@ -77,8 +77,8 @@ neard = { path = "./neard"}
 testlib = { path = "./test-utils/testlib" }
 
 [profile.release]
-lto = true        # Enable full link-time optimization.
-codegen-units = 1 # Use only 1 codegen-unit to enable full optimizations.
+# lto = true        # Enable full link-time optimization.
```

Comment:
hm, why is this needed? it's not in the instructions for running param estimator

Reply:
it's not needed, just for compile faster (10s vs 5min with a change in vm_estimator.rs), should be revert in the final commit